README.md

```diff
@@ -159,7 +159,7 @@ import numpy as np
 import onnx
 import os
 import glob
-import onnx_backend as backend
+from onnx.backend.base import Backend as backend
```

Comment:
This was a design decision, so that users could choose which backend they wanted to run onnx models with (i.e. onnxruntime, pytorch, etc.)